lib/sky-pages-route-generator.js

```diff
@@ -142,10 +142,11 @@ function generateDeclarations(routes) {
   return `[\n${declarations}\n]`;
 }
 
-function generateRuntimeImports(routes) {
+function generateRuntimeImports(skyAppConfig, routes) {
+  const alias = skyAppConfig.runtime.spaPathAlias;
   return routes
     .filter(r => r.guard)
-    .map(r => `import { ${r.guard.name} } from '${r.guard.path.replace(/\.ts$/, '')}';`);
+    .map(r =>`import { ${r.guard.name} } from '${alias}/${r.guard.path.replace(/\.ts$/, '')}';`);
```

Comment:
Since you were able to make these code changes without having to update any unit tests, seems you've found a gap in our coverage.  :-( 

Would you be comfortable writing a quick unit test to verify spaPathAlias is appended to the importPath?